core/tests/protractor/collections.js

```diff
@@ -71,6 +75,33 @@ describe('Collections', function() {
     users.logout();
   });
 
+  it('searches explorations in collection editor', function() {
+    users.login('alice@collections.com');
+    browser.get(general.SERVER_URL_PREFIX);
+    var dropdown = element(by.css('.protractor-test-profile-dropdown'));
```

Comment:
Perhaps factor out into general.moveToCreatorDashboard(), now that we're starting to duplicate code?